tests/template_tests/syntax_tests/test_i18n.py

```diff
@@ -435,6 +435,24 @@ def test_i18n38_2(self):
             'fr: French/français/francouzsky bidi=False; '
         )
 
+    # blocktrans tag with asvar
+    @setup({'i18n39': '{% load i18n %}{% blocktrans asvar page_not_found %}Page not found{% endblocktrans %}>{{ page_not_found }}<'})
```

Comment:
Could you follow the style of the other tests and multi-line these strings so don't exceed 119 characters?